src/EditorFeatures/TestUtilities/Rename/RenamerTests.cs

```diff
@@ -29,7 +29,7 @@ protected struct DocumentWithInfo
             public string Text { get; set; }
             public string DocumentName { get; set; }
             public string DocumentFilePath { get; set; }
```

Comment:
❔ Is there any difference between `{ get; set; }` and `{ readonly get; set; }`?

Reply:
yes.  it will allow hte compiler/analyzers to actually validate that the getter does not mutate/copy this.

Reply:
Note that I'm asking about _literally this exact form_, where in all cases the contents of both accessors are compiler-generated.

Reply:
![image](https://user-images.githubusercontent.com/4564579/220191673-5a3d3629-d198-48ea-8ad0-b8820e56a47c.png)

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/struct#readonly-instance-members